tests/foreman/ui/test_contentview.py

```diff
@@ -3867,3 +3867,71 @@ def test_positive_inc_publish_cv(session, module_org):
         )
         cvv = entities.ContentView(id=cv.id).read().version[1].read()
         assert cvv.package_count == 32
+
+
+@pytest.mark.tier2
+def test_positive_able_to_disable_and_enable_rhel_repos(
```

Comment:
I check mentioned BZ is for ContentView but this test does not seem to relate to ContentView and does look more related to Repository component, so is it okay to keep this test here?

Reply:
Agreed.  I was debating this myself.  I have add it to `repository` instead.